### ISSUE TITLE
Training highway-v0 with DQN, the trained result is not as good as the one presented in the example video

### ISSUE THREAD

**Issue by limeng-1234** [eleurent/highway-env#226]
example code  in the highway-env document
import gym
import highway_env
from stable_baselines import DQN
model = DQN('MlpPolicy', "highway-fast-v0",
policy_kwargs=dict(net_arch=[256, 256]),
learning_rate=5e-4,
buffer_size=15000,
learning_starts=200,
batch_size=32,
gamma=0.8,
train_freq=1,
gradient_steps=1,
target_update_interval=50,
verbose=1,
tensorboard_log="highway_dqn/")
model.learn(int(2e4))
model.save("highway_dqn/model")

Question：
1、highway-fast-v0 does not seem to exist
2、the result is not good as example
3、These parameters, such as gradient_steps, target_update_interval, do not exist in stable_baselines.DQN

**Comment by eleurent** [comment]
Duplicate of #225

**Comment by limeng-1234** [comment]
Thank you for the detailed reply, and there is still something wrong

Code: import gym
import highway_env
from stable_baselines3 import DQN


importgymimporthighway_envfromstable_baselines3importDQNmodel=DQN('MlpPolicy',"highway-fast-v0",policy_kwargs=dict(net_arch=[256,256]),learning_rate=5e-4,buffer_size=15000,learning_starts=200,batch_size=32,gamma=0.8,train_freq=1,gradient_steps=1,target_update_interval=50,verbose=1,tensorboard_log="highway_dqn/")model.learn(int(2e4))model.save("highway_dqn/model")# Load and test saved modelmodel=DQN.load("highway_dqn/model")whileTrue:done=Falseobs=env.reset()whilenotdone:action,_states=model.predict(obs,deterministic=True)obs,reward,done,info=env.step(action)env.render()
comment: The environment is not packaged as an 'env' in the last part. And it is will be better if you can provide the correct agent's results through DQN


| |
limeng
|
|
***@***.***
|
签名由网易邮箱大师定制
On 9/9/2021 15:28，Edouard ***@***.***> wrote：

Closed #226.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
